Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2009-01021
Initial Package Version: 0.10.21
Upstream Status: Unknown, but it looks like it has been backported from upstream
Origin: Unknown, found in gentoo and fedora, edited to apply with -p1.
Description: Allows gstreamer's configure to note choke on bison version 2.4.1
> checking bison version 2.4.1 >= 1.875... Argument "^B^D^A" isn't numeric in numeric lt (<) at - line 1.
- run 'autoreconf && libtoolize -f' after applying this, in a clean tree.

--- foobar/common/m4/gst-parser.m4	2008/07/20 19:55:14	1.12
+++ foobar/common/m4/gst-parser.m4	2008/12/15 07:06:26	1.13
@@ -15,10 +15,7 @@
   bison_version=`$BISON_PATH --version | head -n 1 |  sed 's/^[[^0-9]]*//' | sed 's/[[^0-9]]*$//' | cut -d' ' -f1`
   AC_MSG_CHECKING([bison version $bison_version >= $bison_min_version])
 
-  if perl -w <<EOF
-    exit ($bison_version < $bison_min_version) ? 0 : 1;
-EOF 
-  then 
+  if perl -we "exit ((v$bison_version ge v$bison_min_version) ? 0 : 1)"; then 
     AC_MSG_RESULT([yes])
   else 
     AC_MSG_ERROR([no])
